### PR TITLE
Valentin_Unit_Test+Husky+Removing_Files

### PR DESCRIPTION
**Pull request by Atadeno**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+var app = require('express')(),
```
Comment (ericsengithub) [inline app.js]: does this file still do anything?

File: coverage/block-navigation.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+/* eslint-disable */
```
Comment (ericsengithub) [inline coverage/block-navigation.js]: is this boilerplate, or did you code this?

File: coverage/base.css [hunk 3]
```diff
@@ -0,0 +1,224 @@
+body, html {
```
Comment (ericsengithub) [inline coverage/base.css]: is this boilerplate?

Comment (Atadeno) [reply coverage/base.css]: Yes 👍

File: app.js (hunk 1 above)
Comment (Atadeno) [reply app.js]: I'll delete it

**Pull request merged**